test/test.ml

```diff
@@ -11,12 +11,16 @@ let () =
                         test_case "Test syntax And" `Quick Syntax.test_syntax_and;
                         test_case "Test syntax Or" `Quick Syntax.test_syntax_or;
                         test_case "Test syntax Not" `Quick Syntax.test_syntax_not;  
+                        test_case "Test syntax Assign" `Quick Syntax.test_syntax_assign;
                 ];
                 "semantic-tests", [
                         test_case "Test Interpreter" `Quick Semantic.test_eval_stm; 
                         test_case "Gloca Semantic Formation" `Quick Semantic.test_iswf; 
                 ];
                 "cfg-tests", [
                         test_case "Control Flow Graph construction" `Quick ControlFlow.test_cfg;
-                ]
+                ];
+                "lv-analysis", [
+                        (*test_case "GEN set" `Quick Analysis.test_gen;*)
```

Comment:
Missing. Also, manually compute a solution and validate check that it is indeed a fixpoint.